Lil-Peep/src/styles/modules/featured-categories.scss

```diff
@@ -91,30 +91,28 @@
     margin-bottom: 20px;
     margin-top: 20px;
     text-align: center;
+    width: 140px;
 
     .scroll-up-button {
-      background-color:  $color-bg-scroll;
+      background-color:  $view-green;
       border-radius: 3px;
       border-style: none;
-      color: $color-body;
-      display: flex;
-      display: inline-flex;
       height: 40px;
       justify-content: center;
-      width: 160px;
+      width: 90px;
```

Comment:
Properties should be ordered background-color, border-radius, border-style, height, justify-content, text-decoration, width